reinforcement_learning/trainer/common/replay_buffer.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import deque
+import numpy as np
+import random
+
+
+class ReplayBuffer(object):
+    """Replay buffer."""
+
+    def __init__(self, buffer_size):
+        self.buffer_size = buffer_size
+        self.count = 0
+        self.buffer = deque()
+
+    def add(self, s, a, r, t, s2):
```

Comment:
Can we add some param information as of what s, a, r, t, s2 means?

Reply:
fixed in 8cc6ed46c28df30daefc0f1ad6377dd8e00ba02c